application/sitebuilder/build_service.py

```diff
@@ -31,6 +31,7 @@ def request_build():
     build.id = str(uuid.uuid4())
     db.session.add(build)
     db.session.commit()
+    return build
```

Comment:
Do we still need this return value?

We're not using it anywhere at all as far as I can see?

Reply:
It's used by the test. But also, it seems better than returning nothing at all?

Reply:
I don't think it is used by the test - the test just calls `do_it()` without assigning the return value to anything.